hub/cmd/pull/pull.go

```diff
@@ -24,8 +25,21 @@ func NewCommand(hubOpts *hubOptions.HubOptions) *cobra.Command {
 	opts := hubOptions.NewHubPullOptions(hubOpts)
 
 	cmd := &cobra.Command{
-		Use:   "pull <digest>",
+		Use:   "pull <agent_id>",
 		Short: "Pull an agent from Agent Hub",
+		Long: `Pull an agent from the Agent Hub.
+
+Parameters:
+  <agent_id>    Agent identifier in one of the following formats:
```

Comment:
just a note here, for me it makes more sense for this to be called `agent_ref` as it can be resolved to _different_ `agent_id` rather than always having the _same_ `agent_id` when this is the case of specifying via digest only. wdyt?

Reply:
Sounds good. Same for `repository_id`?